src/ts/components/subtitlesettings/settinglist.ts

```diff
@@ -0,0 +1,31 @@
+import {FontColorSelectBox} from './fontcolorselectbox'
+import {FontOpacitySelectBox} from './fontopacityselectbox'
+import {FontFamillySelectBox} from './fontfamillyselectbox'
+import {FontSizeSelectBox} from './fontsizeselectbox'
+import {BackgroundColorSelectBox} from './backgroundcolorselectbox'
+import {BackgroundOpacitySelectBox} from './backgroundopacityselectbox'
+import {WindowColorSelectBox} from './windowcolorselectbox'
+import {WindowOpacitySelectBox} from './windowopacityselectbox'
+import {CharacterEdgeSelectBox} from './characteredgeselectbox'
+import {SettingsPanelItem} from '../settingspanel'
+import {SubtitleOverlay} from '../subtitleoverlay'
+import {SubtitlePanelCloser} from '../subtitlesettingtoggle'
+import {Component, ComponentConfig} from '../component'
+
+// Helper function since the subtitle option panel should not change
+let GetSubtitleSettingList = (overlay: SubtitleOverlay): Component<ComponentConfig>[] => {
+    return [
+        new SettingsPanelItem('Font', new FontColorSelectBox({}, overlay)),
+        new SettingsPanelItem('Font opacity', new FontOpacitySelectBox({}, overlay)),
+        new SettingsPanelItem('Background', new BackgroundColorSelectBox({}, overlay)),
```

Comment:
No, not sub menus for each @Succo. Just reordering them so all `font` settings are below each other, e.g.
1. Font family
1. Font color
1. Font size
1. Font opacity
1. Character Edge Style
1. Background color
1. Background opacity
1. Window color
1. Window opacity

Currently there are two font options at the top of the list, then other settings and at the bottom again a few font options